app/views/users/_form.html.erb

```diff
@@ -8,68 +8,67 @@
     <div class='col-lg-10 col-md-10 col-sm-10' style="display:flex; justify-content: center;">
       <h3> Sign up with </h3>
     </div>
-      <div class='col-lg-10 col-md-10 col-sm-10' style="display:flex; justify-content: center;">
-        <%= render :partial => "layouts/social_icons" %>
-      </div>
-
-   <div class='col-lg-10 col-md-10 col-sm-10' style="display:flex; justify-content: center;">
-    <h3> OR</h3>
 
-   </div>
-   <br style="clear:both;"/>
+    <div class='col-lg-10 col-md-10 col-sm-10' style="display:flex; justify-content: center;">
```

Comment:
Ah i see there are still some changes here, but are you confident the form still works?